prow/cluster/jobs/istio/istio.io/istio.istio.io.master.yaml

```diff
@@ -0,0 +1,16 @@
+presubmits:
+
+  istio/istio.io:
+
+    - name: presubmit-master
```

Comment:
nit: I recommend including the repo in the job name. It is good convention to use globally unique job names so that jobs can be identified by name and so that naming schemes for presubmit/postsubmit/periodic variants of a job don't result in collisions.